src/query/dblink_scan.c

```diff
@@ -275,7 +392,7 @@ SCAN_CODE
 dblink_scan_next (DBLINK_SCAN_INFO * scan_info, val_list_node * val_list)
 {
   T_CCI_ERROR err_buf;
-  int col_no, col_cnt, ind, error;
+  int col_no, col_cnt, ind, error = S_SUCCESS;
```

Comment:
```suggestion
  int col_no, col_cnt, ind; 
  SCAN_CODE error = S_SUCCESS;
```

Reply:
Ok, thanks

Reply:
Then it should be initialized to NO_ERROR.
NO_ERROR is 0 and S_SUCCESS is 1.